containenv/commands/containenv.py

```diff
@@ -1,5 +1,38 @@
-#!/usr/bin/env python
+"""
+Usage: containenv [--verbose] [--debug <LEVEL>] ( init | help ) [<args>...]
```

Comment:
If you take `[--log-level <level>]` as an option you get logging for free from rapcom.